src/lib/transaction_parser.c

```diff
@@ -383,17 +383,8 @@ int16_t transaction_get_display_key_value(char *key, int16_t max_key_length,
                 key_name = "account_number";
                 break;
             case 2:
-                key_name = "sequence";
-                break;
-            case 3:
                 key_name = "memo";
                 break;
-            case 4:
-                key_name = "source";
-                break;
-            case 5:
-                key_name = "data";
-                break;
```

Comment:
Nobody can do anything harmful things on `account_number` too. Maybe we can remove it from preview page too. 
`data` is a reserved field. Currently, it must be nil. Maybe in the future, we will implement improtant features on it. So I think it would be better to keep it.
As for `Source`, it is used to indicate where the tx comes from, such as trustwallet. So I think it would be better to keep it too.
What do you think about this? @darren-liu @notatestuser

Reply:
* what if signing a doc for a different unintended `account_number`?
* unless we know what `data` will be, it's best to leave it out for this version.
* `source` is not necessary, it's low risk metadata.

Reply:
> * what if signing a doc for a different unintended `account_number`?

Once the account number is incorrect, the signature will be invalid and `CheckTx` will be failed.

> * unless we know what `data` will be, it's best to leave it out for this version.
> * `source` is not necessary, it's low risk metadata.

That's OK.

Reply:
> > * what if signing a doc for a different unintended `account_number`?
> 
> Once the account number is incorrect, the signature will be invalid and `CheckTx` will be failed.

True, unless the HD path is changed and the user is still signing with their seed, but with a different address. It's an edge case, but possible. I would say that the consequences of this are limited since the user will still have access to the account so we can go ahead and remove this.